javascript/fetch-repos.js

```diff
@@ -122,6 +122,15 @@
           }));
         }
       });
+
+      FourthWall.fetchDefer({
+        url: team.baseUrl + "/teams/" + teamId + "/members",
+        done: function (result) {
+          FourthWall.importantUsers = result.map(function(item) {
```

Comment:
I think this should extend `impotantUsers` rather than overwrite it. Multiple teams can be provided, for example we have one team set for `github.com` and one for `github.gds`.